Automattic/jetpack
# VideoPress: remove video chapters block

Fixes https://github.com/Automattic/jetpack/issues/28059

#### Changes proposed in this Pull Request:

* VideoPress: remove video chapters block

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Restart your compiling system
* Build Jetpack and VideoPress plugin
* Confirm Video Chapters (beta) block is not available anymore
* Confirm everything works as expected
